tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "rootDir": "."
+  },
+  "extends": "./tsconfig.production.json",
```

Comment:
Why would we need `tsconfig.production.json`? It's not being built, right? As far as I can see we should have just one `tsconfig.json` file and that's it.

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/7f5d5a019937421c05b398aeca35cd4901d9db26